amadeus/shopping/FlightOffers.cs

```diff
@@ -61,7 +61,7 @@ public FlightOffer[] get(Params _params)
         {
             try
             {
-                Response response = client.get("/v1/shopping/flight-offers", _params);
+                Response response = client.get("/v2/shopping/flight-offers", _params);
```

Comment:
The actual and correct API call of the Flight Offers Search API is at this [class](https://github.com/amadeus4dev-examples/amadeus-dotnet/blob/master/amadeus/shopping/FlightOffersSearch.cs). It seems at the file FlightOffers.cs it hasn't been removed from the past.